Source/Common/EnumHelper.cs

```diff
@@ -1,13 +1,17 @@
 namespace Schema.NET
 {
     using System;
+    using System.Diagnostics;
     using System.Diagnostics.CodeAnalysis;
 
     /// <summary>
     /// Helper for parsing strings into Enum values.
     /// </summary>
     internal static class EnumHelper
     {
+        private const int HttpSchemaOrgLength = 18; // equivalent to "http://schema.org/".Length
```

Comment:
Should we move to use the URL Constants with `.Length` or move these to the constants file?

Reply:
Yep - using the constants with `.Length` now. Even if there is a penalty (there "shouldn't" be because our strings were constant), it should be extremely minor in the scheme of things.

Reply:
There was a minor issue using the constant - there is a trailing slash here that the constant doesn't have. I decided to just `+ 1` the length value from the constant rather than duplicate the string etc. I did update the doc for the constants to mention it excludes the trailing slash.